client/my-sites/stats/feedback/modal/index.tsx

```diff
@@ -90,11 +117,22 @@ const FeedbackModal: React.FC< ModalProps > = ( { siteId, onClose } ) => {
 					onChange={ setContent }
 				/>
 				<div className="stats-feedback-modal__button">
+					{ ! isCheckingAbilityToSubmitFeedback && ! isAbleToSubmitFeedback && (
+						<strong>
+							<em>{ translate( 'You may submit additional feedback 24 hours later.' ) }</em>
```

Comment:
I think we should word this so that it sounds like a limitation on our end. Something like:

> Feedback submission is currently limited to one per 24 hours.

This is part of what I was talking about in Slack. We don't want to make it sound like we don't want their feedback. Instead, we can frame it like it's just an API limitation that we could/might address in a future update.